### DISCUSSION TITLE
Tailscale and Streaming from SDR-Trunk

### DISCUSSION THREAD

**Discussion by Skyking-Bravo** [chuot/rdio-scanner#446] [Q&A]
I am fairly new to Tailscale, and I am obviously missing something. I have Tailscale running and can successfully connect to Rdio-Scanner, but I can't stream from SDR-Trunk. I have updated the Rdio-Scanner URL in SDR-Trunk to https://xxx.xxxxxxxx.ts.net:3000. What am I missing? Is it something on the Tailscale side or Rdio side?

**Comment by rdmcvicar** [comment]
A couple thoughts on the rdio-scanner side (I'm a trunk-recorder guy, so I don't have much on SDR-Trunk):
- Is your rdio server http versus https, by chance?
- Do rdio-scanner logs mention any errors?

**Comment by ScannerGrid** [comment]
> I am fairly new to Tailscale, and I am obviously missing something. I have Tailscale running and can successfully connect to Rdio-Scanner, but I can't stream from SDR-Trunk. I have updated the Rdio-Scanner URL in SDR-Trunk to https://xxx.xxxxxxxx.ts.net:3000. What am I missing? Is it something on the Tailscale side or Rdio side?

Change the https to http in the url, also just making sure that period isn't behind your port in the url box for sdr trunk.
http://100.93.106.109:32778
